website_sale_adj/tests/test_website_sale_adj.py

```diff
@@ -74,3 +74,48 @@ def test_03_create_product(self):
         self.assertEqual(product_order_list[0], product_3)
         self.assertEqual(product_order_list[1], product_2)
         self.assertEqual(product_order_list[2], product_1)
+
+    def test_04_compute_order_line_date(self):
+        """
+            This tests check the earliest
+            website_order_line's creation date
+            """
+        self.product_uom = self.env.ref("product.product_uom_unit").id
+
+        # Create Test Product
+        self.product_01 = self.env["product.product"].create(
+            {"name": "Test Product", "type": "product"}
+        )
+
+        # Create Test Partner
+        self.partner_01 = self.env["res.partner"].create({"name": "Test Partner 1"})
+
+        # Create sale order
+        self.sale_order = self.env["sale.order"].create(
+            {"partner_id": self.partner_01.id}
+        )
+
+        # Create Sale Order line
+        line_01 = self.env["sale.order.line"].create(
+            {
+                "product_id": self.product_01.id,
+                "price_unit": 100.00,
+                "product_uom": self.product_uom,
+                "product_uom_qty": 10.0,
+                "order_id": self.sale_order.id,
+            }
+        )
+
+        # Assign website order line from sale order line
+        self.sale_order.write({"website_order_line": [(6, 0, [line_01.id])]})
```

Comment:
Can we add one more line to make sure the "_compute_order_line_date" will assign the create date of first order line?